staging/src/k8s.io/cri-api/pkg/apis/runtime/v1alpha2/api.pb.go

```diff
@@ -6964,6 +6964,395 @@ func (m *ReopenContainerLogResponse) XXX_DiscardUnknown() {
 
 var xxx_messageInfo_ReopenContainerLogResponse proto.InternalMessageInfo
 
+// Common options used for checkpointing and restoring.
```

Comment:
Wondering what's the plan for adding more options in the future? Is this the exhaustive list or plan is to start with some basic options and to add more based on the use cases? Are these options based on options that are available for criu?

Reply:
The list of options is based on my work on Podman. All the features I implemented there I added here as they seem to be the most interesting to begin with. It also depends on what runc/crun can do. One of the things runc can currently do which is not represented here is pre-copy and post-copy migration. Which is nice to have in the future but I do not think it is important to think about it now.

Reply:
Going through your work on adding checkpoint/restore support for Podman helped me put things in perspective and answered my questions. Thanks!